src/test/java/org/openmrs/contrib/qaframework/page/Attachmentspage.java

```diff
@@ -0,0 +1,52 @@
+package org.openmrs.contrib.qaframework.page;
+
+import java.util.List;
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class Attachmentspage extends Page {
+
+    public static final By ATTACHEMENT = By.id("attachments.attachments.visitActions.default");
+    public static final By CLICKORDROPAFILE = By.id("visit-documents-dropzone");
+    public static final By CAPTION = By.xpath("//*[@id=\"att-page-main\"]/div[1]/att-file-upload/div[2]/div/div[2]/textarea");
+    public static final By CLEARFORMS = By.xpath("//*[@id=\"att-page-main\"]/div[1]/att-file-upload/div[2]/div/div[2]/span/button[2]");
+    public static final By UPLOAD = By.xpath("//*[@id=\"att-page-main\"]/div[1]/att-file-upload/div[2]/div/div[2]/span/button[1]");
+    public static final By LISTOFFILE = By.xpath("//*[@id=\"att-page-main\"]/div[2]/att-gallery/div/div");
+
+
+    public Attachmentspage(Page parent) {
+        super(parent);
+    }
+
+    public void ClickOnAttachement() {
+        clickOn(ATTACHEMENT);
+    }
+
+    public void ClickOnClickOrDropAfile(){
```

Comment:
@achilep  thanks this looks good to me, However , got some slight question. Assuming you make this class **ClickOnOrDropAfile** how does that look , then you can change the static file a relative name. its justa suggestion cc @k-joseph

Reply:
Is there any possibility of adding screen shots for pages you have tested, i think this is quite tricky when it comes to testing such pages in ref-app or using standalone